src/sentry/ratelimits/redis.py

```diff
@@ -106,8 +106,11 @@ def is_limited_with_value(
         # Reset Time = next time bucket's start time
         reset_time = _bucket_start_time(_time_bucket(request_time, window) + 1, window)
         try:
-            result = self.client.incr(redis_key)
-            self.client.expire(redis_key, expiration)
+            pipe = self.client.pipeline()
+            pipe.multi()
+            pipe.incr(redis_key)
+            pipe.execute_command("EXPIRE", redis_key, expiration)
```

Comment:
why the change from `expire` to `execute_command("EXPIRE"`?

Reply:
`.expire()` function wasn't implemented for pipeline, but I'm working on a new PR to remove the dependency first: https://github.com/getsentry/sentry-redis-tools/pull/9

Reply:
Since we updated Redis, we now have `.expire()` command. Resolving this comment. Thank you for reviewing.